css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  font-family: 'Brandon Grotesque', sans-serif;
+}
+
+.banner {
+  display: flex;
+  justify-content: center;
+  font-size: 15px;
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  color: #ee6053;
+}
+
+.submission-form {
+  display: flex;
+  justify-content: center;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.button {
+  display: flex;
+  justify-content: right;
+  padding: 30px;
+}
+
+label {
+  font-size: 15px;
+  vertical-align: bottom;
+  line-height: 0.5px;
+  text-indent: -35px;
+}
+
+input,
+select {
+  font-size: 20px;
+  width: 515px;
```

Comment:
Instead of setting `width` individually on each of these, you should set a width on the `container` class you made and center it using `margin: auto`, then you can make these just `width: 100%` That revision will likely simplify your button placement too.